frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -1320,6 +1322,35 @@ function ReaderFooter:addToMainMenu(menu_items)
                     },
                 },
             },
+            {
+                text_func = function()
+                    return T(_("Minimal width (%1%)"), self.settings.progress_bar_min_width)
+                end,
+                enabled_func = function()
+                    return not self.settings.progress_bar_position and not self.settings.disable_progress_bar
+                        and self.settings.all_at_once
+                end,
+                callback = function(touchmenu_instance)
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local items = SpinWidget:new{
+                        width = Screen:getWidth() * 0.6,
+                        value = self.settings.progress_bar_min_width,
+                        value_min = 20,
+                        value_step = 5,
+                        value_hold_step = 20,
+                        value_max = 50,
```

Comment:
Will this limit the progress bar witdh to 50% - and not allow it to take the remaining 80% when I want it to show only 2 items (we need all at once enabled for that, with all items unchecked except two) ?

Reply:
No. This is only minimal bar width.
With 2 items progress bar will be wider than 50%.
![obraz](https://user-images.githubusercontent.com/22982594/72167711-f92bdb00-33cb-11ea-90ad-d9dc230d3eb6.png)


Of course I can increase value_max to 80 if you want :)

Reply:
Oh, ok, thanks :) Then, no need to increase it I guess.